jupyter/nmm-minimax.ipynb

```diff
@@ -36,8 +29,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`memoize(f)` takes a function and checks if the function was already called with the same two main arguments and returns the saved return value. If it is not yet in the Cache, the arguments and it's return values will be saved to the Cache\n",
+    "## Minimax\n",
     "\n",
+    "Der *Minimax* Algorithmus bietet eine einfache Möglichkeit um den perfekten Weg in einem Nullsummenspiel zu berechnen. Um diesen zu finden wird der komplette Spielbaum vollständig per Tiefensuche durchsucht.\n",
+    "Für einen Spielbaum mit der Tiefe `h` und dem Verzweigungsgrad `b` bedeutet das für die Zeit `t` und den Speicher `m`:\n",
+    "<font size=\"4\">\n",
+    "$$ t_{Minimax} \\in \\mathcal{O}(b^h) $$\n",
+    "$$ m_{Minimax} \\in \\mathcal{O}(bh) $$\n",
```

Comment:
glaube ich finde das mit einem Malpunkt schöner: `$$ m_{Minimax} \in \mathcal{O}(b\cdot h) $$`